File: backend-module/eneity/Score.java
@Data
public class Score {
    private int id;
    private int studentId;
    private int examId;
    private int courseId;
    private double regularScore;
    private double examScore;
    private double finalScore;
    private int rankClass;
    private int rankGrade;
    private int isAbsent;
    private int isCheat;
    private String auditStatus;
    private int enteredBy;
    private String reason;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private boolean isDeleted;
}

File: frontend/src/types/score.ts
export interface Score {
  id: number
  studentId: number
  examId: number
  courseId: number
  regularScore: number
  examScore: number
  finalScore: number
  rankClass: number
  rankGrade: number
  isAbsent: number
  isCheat: number
  auditStatus: string
  enteredBy: number
  reason: string | null
  createdAt: string
  updatedAt: string
  isDeleted: boolean
}

export interface ScoreForm {
  id?: number
  studentId: number
  examId: number
  courseId: number
  regularScore: number | null
  examScore: number | null
  isAbsent?: number
  isCheat?: number
  reason?: string
}

export interface ScoreBatchParams {
  examId: number
  classId: number
  file: File
}
